tests/services/test_genre_detector.py

```
@@ -0,0 +1,85 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from src.services.genre_detector import GenreDetector
+
+class TestGenreDetector(unittest.TestCase):
+
+    def setUp(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)